Bigtable/phpunit-conformance.xml.dist

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
We use the term "conformance" for Firestore, so I moved the acceptance tests to match.